.golangci.yml

```
@@ -0,0 +1,71 @@
+        disabled: false
+  
+  gofmt:
+    simplify: true
+  
+  misspell:
+    locale: US
+
+issues:
+  # Disable default excludes to show all issues
+  exclude-use-default: false
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)